Jenkinsfile_CNP

```diff
@@ -0,0 +1,17 @@
+#!groovy
+
+@Library("Infrastructure")
+
+def type = "java"
+def product = "send-letter-service"
+def component = "send-letters"
```

Comment:
I think product and component are concatenated to make a full name. In this case it would be `send-letter-service-send-letters`, which is a bit much. How about changing component to `api`? @rkondratowicz ?

Reply:
ok I will update the component part to api.